### PR TITLE
s4392444 Knee Laterality Classifier

### PR DESCRIPTION
**Pull request by harrykeightley**
Dear Shakes or to the wonderful tutors,

Here is my assignment in which I implemented the Easy Knee classifier.
I misread what you guys meant by documentation so I have also accidentally documented the large majority of my functions and classes.

Kind regards,
Harry

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Some commits were made after 7th 
- Code seem valid for Easy Difficulty
- Commit messages sufficiently details 
- Results seem good and include both qualitative and quantitative
Other comments: overall good but no need to commit model weights in the future. Could use a bit more comments in ipynb

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

Make sure you clean up any leftovers when you switch from .ipy to .py i.e #In[4]

Rember to remove unneeded commented out code 

Keep all your import statements in one place in a file 

Make sure you remove unused imports

File: recognition/Keats-classifiers/assignment.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
```
Comment (shakes76) [inline recognition/Keats-classifiers/assignment.py:1]: Code header block should be added for all scripts/source files, author info, file description etc.

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. remove unnecessary files like checkpoints and model.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request closed without merging**